sechub-server-core/src/main/java/com/daimler/sechub/server/core/AnonymousCheckAliveRestController.java

```diff
@@ -19,9 +20,10 @@
 				name="REST API call",
 				needsRestDoc=true,
 				description="An anonymous user checks if the server is alive and running using the REST API"))
-	@RequestMapping(path = APIConstants.API_ANONYMOUS + "check/alive", method = RequestMethod.HEAD)
-	public void checkAlive() {
-	    /* do nothing here - its just a HEAD request */
+	@RequestMapping(path = APIConstants.API_ANONYMOUS + "check/alive", method = RequestMethod.GET, produces = {MediaType.APPLICATION_JSON_VALUE})
```

Comment:
Please map the `RequestMethod` explicitly to `GET` and `HEAD`: https://stackoverflow.com/a/3803882

`GET` is mapped to `HEAD` implicitly by Spring MVC: https://docs.spring.io/spring-framework/docs/current/spring-framework-reference/web.html#mvc-ann-requestmapping-head-options

Reply:
@4w31ss3 please do as @Jeeppler  suggested: please add both to RequestMethod so it is 
1. it is more clear that we support both
2. api changes will not affect us

Reply:
@de-jcup @Jeeppler done